test/backtrace.jl

```diff
@@ -5,10 +5,29 @@ have_backtrace = false
 for l in bt
     lkup = ccall(:jl_lookup_code_address, Any, (Ptr{Void},), l)
     if lkup[1] == :backtrace
-        @test lkup[4] == false # fromC
+        @test lkup[6] == false # fromC
         have_backtrace = true
         break
     end
 end
 
-@test have_backtrace
+macro issue1334_inlinedcode()
+    :(begin error("foo") end)
+end
+
+let inlinedfile, inlinedline
+    function testinline()
+        @issue1334_inlinedcode
+    end
+
+    try
+        testinline()
+    catch err
```

Comment:
> LoadError("C:\\projects\\julia\\test\\backtrace.jl",18,ErrorException("invalid redefinition of constant err"))

looks like CI wants you to pick a different name here (or add err to the let block variable list)